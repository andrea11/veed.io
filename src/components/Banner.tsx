export default function Banner() {
    return (
        <div
            className="relative isolate flex items-center gap-x-6 overflow-hidden bg-gray-50 py-2.5 px-6 sm:px-3.5 sm:before:flex-1">
            <div className="flex flex-grow items-center">
                <p className="text-sm leading-6 text-gray-900">
                    <strong className="font-semibold">Veed.io</strong>
                    <svg viewBox="0 0 2 2" className="mx-2 inline h-0.5 w-0.5 fill-current" aria-hidden="true">
                        <circle cx={1} cy={1} r={1}/>
                    </svg>
                    Here the Frontend Coding Challenge by <i>Andrea Accardo</i>
                </p>
            </div>
        </div>
    )
}