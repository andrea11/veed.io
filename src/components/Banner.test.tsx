import Banner from "@/components/Banner";
import {render} from "@testing-library/react";

describe("Banner", () => {
    it("renders correctly", () => {
        const { container } = render(<Banner/>)
        expect(container).toMatchSnapshot()
    })
})