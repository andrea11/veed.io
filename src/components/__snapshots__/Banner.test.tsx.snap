// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders correctly 1`] = `
<div>
  <div
    class="relative isolate flex items-center gap-x-6 overflow-hidden bg-gray-50 py-2.5 px-6 sm:px-3.5 sm:before:flex-1"
  >
    <div
      class="flex flex-grow items-center"
    >
      <p
        class="text-sm leading-6 text-gray-900"
      >
        <strong
          class="font-semibold"
        >
          Veed.io
        </strong>
        <svg
          aria-hidden="true"
          class="mx-2 inline h-0.5 w-0.5 fill-current"
          viewBox="0 0 2 2"
        >
          <circle
            cx="1"
            cy="1"
            r="1"
          />
        </svg>
        Here the Frontend Coding Challenge by 
        <i>
          Andrea Accardo
        </i>
      </p>
    </div>
  </div>
</div>
`;
